Well after some days investigate  

With
```set(CoreExtraLibs -Wl,-Bsymbolic)```

```Error:(148) undefined reference to 'ANativeWindow_release'
Error:(1045) undefined reference to 'ANativeWindo´´´

So fix renference needed add android link too.

I.e 
```set(CoreExtraLibs android -Wl,-Bsymbolic)```

The ${CoreExtraLibs} is not mandatory necesary but ok. 

If you have this problem you should apply this pull request to linker correctly under x86_64 android.. sigh

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1617,7 +1617,7 @@ add_library(${CoreLibName} ${CoreLinkType}
 if(ANDROID)
   set(CoreExtraLibs ${CoreExtraLibs} android)
   if(X86_64)
-    set(CoreExtraLibs ${CoreExtraLibs} android -Wl,-Bsymbolic)
+    set(CMAKE_SHARED_LINKER_FLAGS "${CMAKE_SHARED_LINKER_FLAGS} -Wl,-Bsymbolic")
   endif()
 endif()	
  

